ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Ds3Cli.java

```diff
@@ -193,8 +197,17 @@ private CliCommand getCommandExecutor() {
             case GET_PHYSICAL_PLACEMENT: {
                 return new GetPhysicalPlacement(this.ds3Provider, this.fileUtils);
             }
-            case GET_SERVICE:
+            case GET_TAPE_FAILURE: {
+                return new GetTapeFailure(this.ds3Provider, this.fileUtils);
+            }
+            case DELETE_TAPE_FAILURE: {
+                return new DeleteTapeFailure(this.ds3Provider, this.fileUtils);
+            }
+            case GET_SERVICE: {
+                return new GetService(this.ds3Provider, this.fileUtils);
+            }
             default: {
+                System.err.println("Command not recognized");
```

Comment:
I didn't know if there was appetite, but I wondered about registering commands with Ds3Cli and getting rid of the long if then stream. This is a funny error because if you type a bad command it will be caught by the argument parser. This is only hit when you have one defined in CommandType but not implemented. Old behavior was to default to getService() but I can attest to that's not being terribly intuitive.